backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/pporder/maturing/M_Maturing_Configuration_StepDef.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * #%L
+ * de.metas.cucumber
+ * %%
+ * Copyright (C) 2024 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.cucumber.stepdefs.pporder.maturing;
+
+import de.metas.cucumber.stepdefs.DataTableRow;
+import de.metas.cucumber.stepdefs.M_Product_StepDefData;
+import de.metas.cucumber.stepdefs.StepDefDataIdentifier;
+import de.metas.util.Services;
+import io.cucumber.datatable.DataTable;
+import io.cucumber.java.en.Given;
+import lombok.AccessLevel;
+import lombok.NonNull;
+import lombok.experimental.FieldDefaults;
+import org.adempiere.ad.dao.IQueryBL;
+import org.adempiere.model.InterfaceWrapperHelper;
+import org.compiere.model.I_M_Maturing_Configuration;
+import org.compiere.model.I_M_Maturing_Configuration_Line;
+import org.compiere.model.I_M_Product;
+
+import static org.adempiere.model.InterfaceWrapperHelper.saveRecord;
+
+@FieldDefaults(level = AccessLevel.PRIVATE, makeFinal = true)
+public class M_Maturing_Configuration_StepDef
+{
+	M_Maturing_Configuration_StepDefData maturingConfigurationTable;
+	M_Maturing_Configuration_Line_StepDefData maturingConfigurationLineTable;
+	M_Product_StepDefData productStepDefData;
+	IQueryBL queryBL = Services.get(IQueryBL.class);
+
```

Comment:
all of the above: final

Reply:
they already are:
Line42: `@FieldDefaults(level = AccessLevel.PRIVATE, makeFinal = true)`